.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -35,11 +35,14 @@ We will refer to the release that is going out as the *current* release.
 - [ ] Create a branch off of master (DO NOT PUSH YET) for the *current* milestone of format `releases/v2.3` (where 2.3 is the *current* milestone). After that, anything landing in master will be part of the next milestone.
 - [ ] On the Release branch, pin the AC version to the stable version ([example](https://github.com/mozilla-mobile/fenix/commit/e413da29f6a7a7d4a765817a9cd5687abbf27619)) with commit message "Issue #`<this releng issue>`: Pin to stable AC `<version>` for release v2.3" (replacing 2.3 with the version)
 - [ ] Go through the list of issues closed during this sprint and make sure they all have the correct milestone.
+!!! It looks like ^ is for assembling a changelog. If this is correct, could we explicitly say so?
 - [ ] Add `eng:qa:needed` flags on each issue that still needs it.
```

Comment:
This and the point above it will be done at the same time (otherwise we loop through all the issues twice).  Could we reformat to:

- For each issue closed since the last release (!!!explain how that works):
  - [ ] Ensure it has the correct milestone.
  - [ ] Add `eng:qa:needed` flags on each issue that still needs it.

Reply:
I wound up making this suggested change in https://github.com/mozilla-mobile/fenix/pull/6509